Sources/SATSCore/Assets/ThemeColors.xcassets/Platinum Theme/Primary/platinumPrimaryHighlight.colorset/Contents.json

```diff
@@ -5,9 +5,9 @@
         "color-space" : "srgb",
         "components" : {
           "alpha" : "1.000",
-          "blue" : "0.455",
-          "green" : "0.416",
-          "red" : "0.376"
+          "blue" : "0x3C",
```

Comment:
Is this color correct? In the design system I find hex color 161718 for Primary Platinum Hover.

Reply:
this was updated after as well

Reply:
In figma? I see that the correct hex code for this is now: 313336.

Reply:
that's the value this color has now. I asked Magnus to update it, before it was the same as the normal primary.

Reply:
I see.